console/src/main/webapp/WEB-INF/spring/rabbit-listener-context.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:rabbit="http://www.springframework.org/schema/rabbit" xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.1.xsd
+http://www.springframework.org/schema/rabbit http://www.springframework.org/schema/rabbit/spring-rabbit-1.0.xsd">
+    <rabbit:connection-factory id="connectionFactory" host="rabbitmq" username="user" password="bitnami" />
```

Comment:
to be configured

Reply:
> yes, please make host/usr/pwd configurable in {georschestra.datadir}/console/console.properties

done in default.properties as rabbitmq is used by more than webservice.